packages/principles-core/src/runtime-v2/feedback/__tests__/redact-sensitive.test.ts

```
@@ -0,0 +1,525 @@
+
+  it('preserves non-sensitive fields', () => {
+    const obj = { name: 'test', count: 42, enabled: true, data: null };
+    const result = redactSensitiveFields(obj);
+    expect(result.ok).toBe(true);
+    if (result.ok) {
+      expect(result.value).toEqual({ name: 'test', count: 42, enabled: true, data: null });
+    }
+  });
+
+  it('handles BigInt values', () => {
+    const obj = { big: BigInt(12345678901234567890) };
```

ESLint

[error] This number literal will lose precision at runtime.

(no-loss-of-precision)